doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1220 @@
+      For further details on the Android intent feature for firmware updates, see the :ref:`ug_bt_fast_pair_provisioning_register_firmware_update_intent` section in the Fast Pair integration guide.
+    * A workaround for the issue where the FMDN clock value might not be correctly set after the system reboot for nRF54L Series devices.
+      For details, see the NCSDK-32268 known issue in the :ref:`known_issues` page.
+    * A new function :c:func:`bt_fast_pair_fmdn_is_provisioned` for the FMDN extension API.
+      This function can be used to synchronously check the current FMDN provisioning state.
+      For more details, see the :ref:`ug_bt_fast_pair_gatt_service_fmdn_info_callbacks_provisioning_state` section in the Fast Pair integration guide.
+
+  * Updated:
+
+    * The :c:func:`bt_fast_pair_info_cb_register` API to allow registration of multiple callbacks.
+    * The Fast Pair sysbuild Kconfig options.
+      The ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option is replaced with the ``SB_CONFIG_BT_FAST_PAIR_MODEL_ID`` and ``SB_CONFIG_BT_FAST_PAIR_ANTI_SPOOFING_PRIVATE_KEY``.
+    * The method of supplying the Fast Pair Model ID and Anti-Spoofing Private Key to generate the Fast Pair provisioning data HEX file.
+      The ``FP_MODEL_ID`` and ``FP_ANTI_SPOOFING_KEY`` CMake variables are replaced by the corresponding ``SB_CONFIG_BT_FAST_PAIR_MODEL_ID`` and ``SB_CONFIG_BT_FAST_PAIR_ANTI_SPOOFING_PRIVATE_KEY`` Kconfig options.
+    * The automatically generated ``bt_fast_pair`` partition definition (in the :file:`subsys/partition_manager/pm.yml.bt_fast_pair` file) to work correctly when building with TF-M.
+    * The behavior of the :c:member:`bt_fast_pair_fmdn_info_cb.provisioning_state_changed` callback.
+      The callback no longer reports the initial provisioning state after the Fast Pair subsystem is enabled with the :c:func:`bt_fast_pair_enable` function call.
+      See the :ref:`migration guide <migration_3.0_recommended>` for mandatory changes and the :ref:`ug_bt_fast_pair_gatt_service_fmdn_info_callbacks_provisioning_state` section in the Fast Pair integration guide for the description on how to track the FMDN provisioning state with the new approach.
+
+  * Removed the sysbuild control over the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option that is defined in the main (default) image.
+    Sysbuild no longer sets the value of this Kconfig option.
+
+* :ref:`bt_mesh` library:
+
+  * Fixed an issue in the :ref:`bt_mesh_light_ctrl_srv_readme` model to automatically resume the Lightness Controller after recalling a scene (NCSDK-30033 known issue).
+
+Common Application Framework
+----------------------------
+
+* :ref:`caf_buttons`:
+
+  * Added the possibility of using more GPIOs.
+    Earlier, only **GPIO0** and **GPIO1** devices were supported.
+    Now, the generic solution supports all GPIOs available in the DTS.
+
+* :ref:`caf_power_manager`:
+
+  * Updated:
```

Comment:
Please add under `caf_power_manager` -> Updated:

```
  * The module implementation to use :c:func:`sys_poweroff` API instead of :c:func:`pm_state_force` API to enter the system off state.
  * The module implementaion to integrate the :c:struct:`power_off_event`.
```

Reply:
Added.